docs/guide/extending/extending_back_office.md

```diff
@@ -0,0 +1,231 @@
+# Extending Back Office
+
+## Custom Content Type icons
+
+To add custom icons for existing Content Types or custom Content Types in eZ Platform follow the instructions below.
+
+### Configuration
+
+A configuration of the default icon for Content Type is possible via `default-config` key.
+For example:
+
+```yaml
+ezpublish:
+    system:
+       default:
+           content_type:
+              default-config:
+                 thumbnail: /assets/images/mydefaulticon.svg
+```
+
+To configure a custom icon you just need to replace the `default-config` key with a Content Type identifier.
+For example:
```

Comment:
If you decide to apply the suggestion above (For example: -> e.g.:), please apply it to the whole section.